### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by stefaner1** [docker/compose-on-kubernetes#158]
When trying to install it on Ubuntu with microk8s I try to use this command, but it doesn't work:
` ./installer-[darwin|linux|windows.exe] -namespace=compose -etcd-servers=http://compose-etcd-client:2379`

This doesn't work too:
`./installer-linux -namespace=compose -etcd-servers=http://compose-etcd-client:2379`

Do I miss sth?

**Comment by ebeltran1981** [comment]
@stefaner1 I am having the same issue. Looking at the compose/compose-api pods, they are logging an error: `W0318 20:36:44.333026       1 clientconn.go:1120] grpc: addrConn.createTransport failed to connect to {http://compose-etcd-client:2379 0  <nil>}. Err :connection error: desc = "transport: Error while dialing dial tcp: lookup compose-etcd-client on 10.152.183.10:53: no such host". Reconnecting...`

I've tried replacing the `compose-etcd-client` with localhost or 127.0.0.1 and doesnt work either. 

did you figure it out?

**Comment by stefaner1** [comment]
They fixed It here: #151

`Run installer-[darwin|linux|windows.exe] --namespace=compose --etcd-servers=http://compose-etcd-client:2379`

But it looks like it all buggy. I couldn't make it to the end of installation.

**Comment by ebeltran1981** [comment]
@stefaner1 I got it figure it out.

[here](https://github.com/docker/compose-on-kubernetes/blob/master/docs/install-on-microk8s.md#deploy-etcd), in the instructions to install etcd, the [Option 1](https://github.com/docker/compose-on-kubernetes/blob/master/docs/deploy-etcd.md#option-1-create-an-etcd-cluster-for-quick-evaluation) says is only for evaluation; apparently that is the service the `http://compose-etcd-client:2379` is trying to hit. After I did that, it started working.

![image](https://user-images.githubusercontent.com/1435275/77072955-de29a880-69c4-11ea-905e-7e8e8c7e6723.png)

This is my setup:

- Ubuntu Server 18.04
- microk8s 1.14.10
- helm 2.16.3
- `sudo apt-get install iptables-persistent` to make iptables forward policies persistent, then `sudo iptables -P FORWARD ACCEPT`
- run `microk8s.inspect` to make sure everything is fine.
- then follow the instructions from the compose on kubernetes page

hope this helps.

Now I will try to deploy a compose using docker 😅
